packages/core/src/LegalLinks/index.js

```diff
@@ -28,7 +28,7 @@ const useStyles = makeStyles((theme) => ({
     },
     [theme.breakpoints.up("md")]: {
       "& > li": {
-        borderRight: `1px solid ${theme.palette.text.secondary}`,
+        // borderRight: `1px solid ${theme.palette.text.secondary}`,
```

Comment:
Commented code

Reply:
@gertie-sheshe So for this, I reverted it back to how it was just to stick with the convention we are using `Material ui` after yesterday's morning call.